launcher/minecraft/mod/ModFolderModel.h

```diff
@@ -64,6 +64,7 @@ class ModFolderModel : public ResourceFolderModel
     enum Columns
     {
         ActiveColumn = 0,
+        ModUpdateColumn,
```

Comment:
i think this column could be moved to the right, especially as it's not really that important compared to the other columns i think